recipes/opportunity_insights/create_weekly.sql

```diff
@@ -21,15 +23,29 @@ SELECT
             THEN 'NYC'
         ELSE 'Region'
     END) as location,
-    county_name as county,
+    SPLIT_PART(county_name, ' County', 1) as county,
+    (CASE
+        WHEN fipscounty ~* '9' THEN 'CT'
```

Comment:
The following should be assigned in NJ instead
![image](https://user-images.githubusercontent.com/13207770/93616252-80d9d880-f9a2-11ea-9d50-e18c06e2e7b8.png)

We should check the first two numbers
`WHEN LEFT(fipscounty, 2) = '09' THEN 'CT'`